### PR TITLE
Small fix to multicontainer job: change bash to sh

### PR DESCRIPTION
**Pull request by anthonyrtong**
Using /bin/sh instead of /bin/bash for job since alpine image does not have bash.

### CODE REVIEW

**Comment by BenTheElder** [comment]
Does this really need to be running every 10m?
Can you set explicit (low) resource requests?

**Comment by anthonyrtong** [comment]
This doesn't need to run every 10m.  I'll revise this to be hourly.

File: config/jobs/kubernetes/test-infra/test-infra-canaries.yaml [hunk 1]
```diff
@@ -43,18 +43,30 @@ periodics:
       image: alpine
       command: ["/bin/echo"]
       args: ["I am first"]
+      resources:
+        requests:
+          memory: "128Mi"
```
[the remark this answers is not in the document]
Comment (anthonyrtong) [reply config/jobs/kubernetes/test-infra/test-infra-canaries.yaml:48]: Yep, I added some limits.

**Pull request merged**